zkevm-circuits/src/pi_circuit.rs

```diff
@@ -478,8 +499,24 @@ impl<F: Field> SubCircuitConfig<F> for PiCircuitConfig<F> {
             ]
         });
 
+        meta.create_gate("withdrawal id is incremental", |meta| {
```

Comment:
For withdrawl id, because each table value is copy constraint from public input, so the id format should assumed to be valid, so maybe we do not need to add new gate here. 

(similar situation for gate "tx_id_diff")

I think they are some left over due to previously public input circuit are using rlc constriant. But now we have copy constraints, so each cell value is more accurate

Reply:
Based on above description, we do not need `q_wd_table` as well, due to it's only used for "withdrawal id is incremental" gate

Reply:
removed the constraint and `q_wd_table` in [277a031](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1634/commits/277a031c77fe8961fb7b84719570c4b33012596d)